src/lib/pmd/PmdEngine.ts

```diff
@@ -260,7 +259,7 @@ abstract class BasePmdEngine implements RuleEngine {
 						column: v.attributes.begincolumn,
 						endLine: v.attributes.endline,
 						endColumn: v.attributes.endcolumn,
-						severity: v.attributes.priority,
+						severity: Number(v.attributes.priority),
```

Comment:
What necessitated this?

Reply:
The severity was being stored as a string.